cookbooks/bcpc-hadoop/recipes/hdfs_site.rb

```diff
@@ -211,13 +211,13 @@
       jn_properties =
         {
          'dfs.journalnode.rpc-address' =>
-           node[:bcpc][:floating][:ip] + ':8485',
+           node[:bcpc][:floating][:ip] + ':' + node['bcpc']['hadoop']['journalnode']['rpc']['port'].to_s,
```

Comment:
For this and other concatenated strings below, could we use interpolation instead? It's arguably *slightly* easier to read, the `.to_s` happens implicitly, and it's also slightly faster in benchmarks (interpolation is faster already, plus it now becomes a single string).

Example:
``` ruby
"#{node['bcpc']['floating']['ip']}:#{node['bcpc']['hadoop']['journalnode']['rpc']['port']}",
```